src/cobalt_strike.yml

```diff
@@ -52,3 +52,7 @@
       vars:
         # Group ownership for the Cobalt Strike addons
         group: wheel
+    - name: Install LibreOffice
+      ansible.builtin.package:
+        name:
+          - libreoffice
```

Comment:
A case could be made that `cobalt_strike.yml` is no longer a great name for this playbook given the variety of tools it now installs, but that certainly doesn't need to be addressed at this moment.

Reply:
Feel free to create an issue to address this, uh, issue.

Reply:
#48 has been issued to address this issue.